cluster/examples/kubernetes/ceph/read-affinity-config/data/3az-template-rule.txt

```diff
@@ -0,0 +1,16 @@
+rule replicated_affinity3_<<AZ1>><<SFX>> {
+    id <<ID>>
+	type replicated   
+	min_size 1
+	max_size 10
+	step take <<AZ1>>
+	step chooseleaf firstn 1 type <<TYP>>
```

Comment:
```suggestion
	step chooseleaf firstn 1 type <<TYPE>>
```

Reply:
This is just a placeholder for the template that is later replaced with the correct type by the script. I can change it, but since all other placeholders are of the structure <<XYZ>> I thought keeping this with the same size as other replcement strings